test/viewselector.spec.js

```diff
@@ -329,23 +305,6 @@ describe('A ViewSelector Component', function () {
 
     });
 
-    it('maintains a list of top level article section headings', () => {
```

Comment:
We might want to raise a follow up ticket just to ensure that this test is covered somewhere else. If the other tests are only checking if the menu has the default options and not the section headings links then it is possible we might miss a break further down the line.